.swagger-revision

```diff
@@ -1 +1 @@
-62fbcb9e8bec4a40569e51c7f316af85806628ee
+v2.2.0
```

Comment:
I've done a bunch of things to try and force this to use a newer version of swagger. We are at least pinned to that tag. It seems like something has to change in https://github.com/DocRaptor/docraptor-swagger/blob/master/script/swagger, probably checking the tag with `git describe --abbrev=0` or something along those lines